lib/evss/common_service.rb

```diff
@@ -7,6 +7,11 @@ class CommonService < BaseService
     API_VERSION = Settings.evss.versions.common
     BASE_URL = "#{Settings.evss.url}/wss-common-services-web-#{API_VERSION}/rest/"
 
+    def initialize(*args)
+      super
+      @use_mock = Settings.evss.mock_rating_info || false
```

Comment:
Looks straight forward. 👍As a suggestion to keep the code more uniform, wondering if the conn method in EVSS base_service.rb should be moved to a configuration class instead

Reply:
Why are we modify the common service? I think these settings belong in inherited class like what is done in VSO search and reference data.

Reply:
I agree with Charlie, there is a lot of critical code that depends on this common service object so something like this should be done much higher up the chain if at all possible.

Reply:
Thanks @charleystran and @bastosmichael.  My original PR had its own custom service and configuration classes (similar to VSO search), but I was directed to use common service since the base URL and configuration already lived there, and I was just adding another endpoint.  However, the mock configuration is not enabled for common services, so I'm not able to use betamocks locally to test the new endpoint.  If someone could give me direction on how to just enable betamocks without duplicating, I'm open to suggestions.  @johnpaulashenfelter tagging you as well for visibility so we can discuss further.

Reply:
The issue I see is that all requests to common service will be mocked or not mocked based on the rating info flag. We do want the dev servers to still use betamocks even though we have some local access now. It may make sense to change your flag to be to mock_common_service vs mock_rating_info

Reply:
@bastosmichael @charleystran I was going to mention the same thing. @omgitsbillryan had asked her to do this: https://github.com/department-of-veterans-affairs/vets-api/pull/3446

Reply:
A closer look informed my last comments. Common is not shared like other common services. Naming is fun. However I still recommend that the mock flag be to the whole service and not the endpoint.

Reply:
Also one interesting note is that vso_search also is a common service so we do have a pattern to break these into individual services.

Reply:
Thank you all for your comments.  I went ahead and updated the settings flag to "mock_common_service" as requested by @charleystran

Reply:
> I still recommend that the mock flag be to the whole service and not the endpoint

Agree with this ☝️. I'd define "service" as a collection of "endpoints" that share configuration. So we have to pick our poison - either we mock a single endpoint in a service or we define a new service along with new duplicated config.

I think we've run up into some tech debt here. This `CommonService` doesn't use our `Configuration` model. Also, there doesn't seem to be anything at all "common" about this service.